src/Utilities/PaymentMethodCollectUtils.res

```diff
@@ -212,33 +210,60 @@ let getPaymentMethodDataFieldPlaceholder = (key: paymentMethodDataField): string
   | SepaCountryCode => "Country"
   | PixId => "**** 3251"
   | PixBankAccountNumber => "**** 1232"
-
   | ACHBankName
   | BacsBankName
   | PixBankName
   | SepaBankName => "Bank Name"
-
   | ACHBankCity
   | BacsBankCity
   | SepaBankCity => "Bank City"
-
   | PaypalMail => "Your Email"
   | PaypalMobNumber | VenmoMobNumber => "Your Phone"
+  | CardBrand => "Misc."
```

Comment:
?

Reply:
This is a meta field, I wanna persist the value whenever it is re-computed (on cardNumber change) in the dict

Since I'm not rendering this as a field in the UI, the return value doesn't really matter (for labels, inputType, maxLength, etc ..)